include/circt/Dialect/SV/SVStatements.td

```diff
@@ -307,7 +319,8 @@ def CaseOp : SVOp<"case", [SingleBlock, NoTerminator, NoRegionArguments,
   let regions = (region VariadicRegion<SizedRegion<1>>:$caseRegions);
   let arguments = (ins DefaultValuedAttr<CaseStmtTypeAttr,
                                          "CaseStmtType::CaseStmt">:$caseStyle,
-                       HWIntegerType:$cond, ArrayAttr:$casePatterns);
+                       HWIntegerType:$cond, ArrayAttr:$casePatterns,
+                       OptionalAttr<ValidationQualifierTypeAttr>: $validationQualifier);
```

Comment:
Instead of an optional attr, make this a defaulted attr and add a "plain" case to the qualifier list.